### PR TITLE
Make csi driver configurable to be able to run multiple instances

### PR DESCRIPTION
**Pull request by kfox1111**
This patch makes it possible to configure the name of the csi driver.

### CODE REVIEW

File: charts/spire/charts/spiffe-csi-driver/values.yaml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# -- Set the csi driver name deployed to Kubernetes.
+pluginName: csi.spiffe.io
```
Comment (faisal-memon) [inline charts/spire/charts/spiffe-csi-driver/values.yaml:2]: Should this be `{{ .Values.pluginName }}`?

Comment (marcofranssen) [reply charts/spire/charts/spiffe-csi-driver/values.yaml:2]: No in values file this should be the real value. Templates only go inside the templates folder.

File: charts/spire/charts/spiffe-csi-driver/values.yaml [hunk 2]
```diff
@@ -15,6 +18,9 @@ resources: {}
   #   cpu: 100m
   #   memory: 64Mi
 
+healthChecks:
+    port: 9809
```
Comment (marcofranssen) [inline charts/spire/charts/spiffe-csi-driver/values.yaml:22]:
```suggestion
  port: 9809
```

**Review by marcofranssen — CHANGES_REQUESTED** [review]
Small nitpick regarding yaml indentation

**Pull request merged**